html/semantics/interactive-elements/the-dialog-element/WEB_FEATURES.yml

```diff
@@ -22,3 +22,6 @@ features:
   - modal-dialog-backdrop-opacity.html
   - modal-dialog-display-contents.html
   - modal-dialog-generated-content.html
+- name: open-pseudo
+  files:
+  - dialog-open-pseudo-invalidation.html
```

Comment:
Noting that I followed the convention already in place in this yml (`dialog` applies to all the tests in this subdirectory). This line _adds_ the `open-pseudo` classifier to a test that is also mapped to `dialog`. 

Not sure if that's what we want to do here. See related question in a [PR mapping the button element](https://github.com/web-platform-tests/wpt/pull/56727#discussion_r2621913579).